documentation/assembly-pdf.xml

```diff
@@ -11,16 +11,6 @@
       <includes>
         <include>*.pdf</include>
```

Comment:
Now that I think of it: how many files are we including, just 1? Maybe we don't need this assembly at all?

Reply:
For pdf, there is one file. For HTML version there are 3. We can remove the assembly for pdf, but don't you think that it is better to have the common mechanism for distributing docs for HTML and pdf?

Reply:
I'm fine either way.